docs/cli-spec.md

```diff
@@ -0,0 +1,40 @@
+## Polystat usages
+
+### Notation
+`|` - exclusive or
+`(...)` - must be present
+`[...]` - may not be present
+`<thing>...` -- multiple instances of thing can be accepted.
+```
+polystat (eo | java | python) --tmp <path> --in <path> [--include <rule>... | --exlude <rule>...] --sarif  --files [<path>]
+polystat --version --help
+polystat list
+```
```

Comment:
I thought about this. It would probably be best to employ some configuration format and make it so that it's possible to specify only certain options. We can use [ciris](https://cir.is/) to populate a Config case class from some of the supported formats.